katana_gripper:
 # Position Controllers ---------------------------------------
  katana_r_finger_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: katana_r_finger_joint
    pid: {p: 0.5, d: 0.1, i: 0.0, i_clamp: 0.0}

  katana_l_finger_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: katana_l_finger_joint
    pid: {p: 0.5, d: 0.1, i: 0.0, i_clamp: 0.0}
